asm/non_matchings/code_80091750/func_800A34A8.s

```diff
@@ -56,9 +56,9 @@ glabel func_800A34A8
 .L800A357C:
 /* 0A417C 800A357C 0C028E84 */  jal   func_800A3A10
 /* 0A4180 800A3580 27A40080 */   addiu $a0, $sp, 0x80
-/* 0A4184 800A3584 3C048019 */  lui   $a0, %hi(gGPOverallRanks) # $a0, 0x8019
+/* 0A4184 800A3584 3C048019 */  lui   $a0, %hi(gCharacterIdByGPOverallRank) # $a0, 0x8019
```

Comment:
This one is very confusing. I don't have a better alternative though.

Reply:
I honestly don't like the name very much either, but I'm unable to find a more terse way of getting across what the array tracks (character IDs) and how it's indexed (overall rank in the current GP)